vonk/features/terminologyintegration.rst

```diff
@@ -0,0 +1,100 @@
+.. _feature_terminologyintegration:
+
+External terminology services
+=============================
+
+.. warning:: The TerminologyIntegration plugin does not work together with the Vonk Terminology plugin. 
```

Comment:
I would like the warning to come after the list of external services, so we focus on what is possible first.